src/languages/jboss-cli.js

```diff
@@ -0,0 +1,67 @@
+/*
+ Language: jboss-cli
+ Author: Raphaël Parrëe <rparree@edc4it.com>
+ Description: language definition jboss cli
+ Category: config
+ */
+
+function (hljs) {
+  var PARAMS = [{
+    className: "attribute",
+    begin: /[\w-]+\s?=/,
+    starts: {
+
+      ends: /$/,
+      className: "string"
+    },
+    relevance: 1
```

Comment:
1 is the default value, so shouldn't be set explicitly (JavaScript being a weird language where we care about code size). Same for all the other occurrences of it down the file.